okio/src/jvmTest/kotlin/okio/utilJvm.kt

```diff
@@ -15,24 +15,26 @@
  */
 package okio
 
+import okio.Path.Companion.toOkioPath
 import kotlin.test.assertEquals
 import kotlin.test.assertFailsWith
-import okio.Path.Companion.toOkioPath
 
 @ExperimentalFileSystem
 actual fun assertRelativeTo(
   a: Path,
   b: Path,
   bRelativeToA: Path,
-  confirmResolutionInversion: Boolean,
-  consistentWithJavaNioPath: Boolean,
+  sameAsNio: Boolean,
 ) {
-  assertEquals(bRelativeToA, b.relativeTo(a))
-  if (confirmResolutionInversion) {
-    assertEquals(b, a / b.relativeTo(a))
+  val actual = b.relativeTo(a)
+  assertEquals(bRelativeToA, actual)
+  try {
+    assertEquals(b.withUnixSlashes(), (a / actual).withUnixSlashes())
+  } catch (e: IllegalArgumentException) {
+    // This is also okay. It means we lose information and can't reverse the operation.
```

Comment:
When does that happen? All were passing on my end, except for the slash difference

Reply:
I just removed the boolean parameter.

Reply:
But now that we do `withUnixSlashes()`, the assertion should never fail!

Reply:
Sorry I misspoke. The information we don’t have is the directory the `..` erases.